src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1896,6 +1896,10 @@ internal enum ErrorCode
         ERR_CannotUseManagedTypeInUnmanagedCallersOnly = 8894,
         ERR_UnmanagedCallersOnlyMethodOrTypeCannotBeGeneric = 8895,
         ERR_UnmanagedCallersOnlyRequiresStatic = 8896,
+        ERR_UnmanagedCallersOnlyMethodsCannotBeCalledDirectly = 8897,
+        ERR_UnmanagedCallersOnlyMethodsCannotBeConvertedToDelegate = 8898,
+        ERR_EntryPointCannotBeUnmanagedCallersOnly = 8890,
+        ERR_ModuleInitializerCannotBeUnmanagedCallersOnly = 8900,
```

Comment:
>8900 [](start = 60, length = 4)

8891?

Reply:
No, the one before it is misnumbered.

---
In reply to: [482588206](https://github.com/dotnet/roslyn/pull/47311#discussion_r482588206) [](ancestors = 482588206)